configure.ac

```diff
@@ -3371,12 +3379,12 @@ AS_VAR_IF([have_uuid], [missing], [
       AC_DEFINE([HAVE_UUID_H], [1])
       AC_DEFINE([HAVE_UUID_GENERATE_TIME_SAFE], [1])
     ], [
-      AC_CHECK_HEADERS([uuid/uuid.h], [
-        WITH_SAVE_ENV(
-          [AC_CHECK_LIB([uuid], [uuid_generate_time], [have_uuid=yes])
-        ])
-        WITH_SAVE_ENV([
-          AC_CHECK_LIB([uuid], [uuid_generate_time_safe], [
+      WITH_SAVE_ENV([
+        CFLAGS="$CFLAGS $LIBUUID_CFLAGS"
```

Comment:
Would it make sense to consistently use CPPFLAGS for these checks? IIRC, configure on FreeBSD had some issues with CFLAGS in one of my sqlite3 PRs.

Reply:
done